### ISSUE TITLE
QGIS 3.18.2 and multilayer-select

### ISSUE THREAD

**Issue by apiszcz** [YoannQDQ/qgis-multilayer-select#3]
![20210425062606](https://user-images.githubusercontent.com/1430861/115990567-f2f64980-a591-11eb-82d5-866df8b9de10.gif)
Thank you for the plugin.
1. A bounding box option would be nice.
2. I am getting a stack trace when selecting settings

```
2021-04-25T06:28:15     WARNING    Traceback (most recent call last):
              File "C:/Users/u1/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\multilayerselect\settingsdialogimpl.py", line 93, in data
              return icon_from_layer(layer)
              File "C:/Users/u1/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\multilayerselect\utils.py", line 79, in icon_from_layer
              index = treemodel.node2index(layer_node)
             AttributeError: 'QSortFilterProxyModel' object has no attribute 'node2index'
```

**Comment by hamiltoncj** [comment]
I am getting that stack trace error as well.

**Comment by YoannQDQ** [comment]
@apiszcz What do you mean by bounding box option?
The node2index error will be fixed once the new version (1.4.1) is approved
